src/components/detailViewChild.js

```diff
@@ -26,8 +24,11 @@
       Body2: 'p',
     }[options.type || 'Body1'];
 
-    const parsedContent = labelIsEmpty ? propertyLabel : useText(labelText);
-    const propName = isDev ? `{{ ${propertyName} }}` : useText([property]);
+    const parsedContent = isDev
+      ? `{{ ${propertyLabel} }}`
+      : useText([{ ...property, type: 'PROPERTY_LABEL' }]);
+    const customLabel = useText(labelText);
+    const propName = isDev ? `{{ ${propertyLabel} }}` : useText([property]);
```

Comment:
FYI @SanderVanRiessen You're conditionally rendering the useText hook here, which is against the rules of hooks. Could be interesting to refactor this at some point.